lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -60,6 +60,15 @@ div.course-wrapper {
       }
     }
 
+    .required-content-msg {
+      margin-bottom: 20px;
+      background-color: $gray-l4;
+      color: $black-t2;
+      font-size: 13px;
+      padding: 5px 10px;
+      margin-top: -20px;
```

Comment:
In general we try to avoid negative margin rules and instead apply this to parent elements, but if you do need to apply this, I would use : -($baseline);